server-utils/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+//! JSON-RPC servers utilities.
+#![deny(missing_docs)]
+
+#[macro_use]
```

Comment:
Let's properly rewrite to `edition=2018` as stated in the toml and remove this stuff.